etc/nixos/configuration.nix

```diff
@@ -0,0 +1,67 @@
+# Edit this configuration file to define what should be installed on
+# your system.  Help is available in the configuration.nix(5) man page
+# and in the NixOS manual (accessible by running ‘nixos-help’).
+
+{ config, pkgs, ... }:
+let
+  vars = import ./vars.nix;
```

Comment:
LOL. I already fixed this on this branch before you commented, possibly before I sent you the text back. You should refresh your PRs.